app/_src/guides/customizing-transparent-proxy-configuration-in-kubernetes-mode.md

```diff
@@ -0,0 +1,434 @@
+---
+title: Customizing Transparent Proxy Configuration in Kubernetes Mode
```

Comment:
In general this feels more like a reference doc not a guide. I'd say a guide has some specific task in mind and explains how to do a thing.

Reply:
Where do you think it should be placed? Under the reference section in the sidebar?